redbpf-tools/build.rs

```diff
@@ -4,6 +4,8 @@ use std::path::{Path, PathBuf};
 use cargo_bpf_lib as cargo_bpf;
 
 fn main() {
+    println!("cargo:rerun-if-changed=probes/");
```

Comment:
This is unnecessary, at the bottom we're already printing a rerun-if-changed for each probe file